src/watts/plugin_pyarc.py

```diff
@@ -45,25 +45,16 @@ def __init__(self, params: Parameters, time: datetime,
     def stdout(self) -> str:
         return (self.base_path / "PyARC_log.txt").read_text()
 
-    def _save_PyARC(self, user_object) -> dict:
+    def _save_PyARC(self, results_data) -> dict:
         """Return PyARC results in a dictionary
 
         Returns
         -------
         Results from PyARC results
 
         """
-        results_data = {}
-        results_data["mcc3"] = user_object.results_keff_mcc3
-        results_data["dif3d"] = user_object.results_keff_dif3d
-        results_data["proteus_nodal"] = user_object.results_keff_nodal
-        results_data["rzmflx"] = user_object.results_keff_rzmflx
-        results_data["persent_pert"] = user_object.results_kpert_persent
-        results_data["persent_sens"] = user_object.results_kuq_persent
-        results_data["gamsor"] = user_object.results_power_gamsor
-        results_data["dassh"] = user_object.results_dassh
-        # TODO - this is minimum information that can be brought back easily.
-        # More should be returned but will require work on PyARC.
+
+        # TODO: need to provide results_data as hdf5 file and extract the information here using _from_hdf5
```

Comment:
This method doesn't seem to have a purpose at this point. Should we just remove it? Even once we add the reading/writing of information to/from HDF5, that will happen in the `save` and `_from_hdf5` methods rather than here.

Reply:
Yes, I removed it. Thanks!